uns_plugin/schema_validation/redpanda_integration_test.go

```
@@ -0,0 +1,728 @@
+	req.Header.Set("Content-Type", "application/vnd.schemaregistry.v1+json")
+
+	resp, err := c.httpClient.Do(req)
+	if err != nil {
+		return fmt.Errorf("failed to register schema: %w", err)
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != 200 && resp.StatusCode != 201 {
+		// Read response body for debugging
+		bodyBytes := make([]byte, 1024)
+		resp.Body.Read(bodyBytes)
```

golangci-lint (1.64.8)

Error return value of `resp.Body.Read` is not checked

(errcheck)